Formula/afl-fuzz.rb

```diff
@@ -11,6 +11,11 @@ class AflFuzz < Formula
   end
 
   def install
+    # test_build dies with "Oops, the instrumentation does not seem to be
+    # behaving correctly!" in a nested login shell.
+    # Reported to lcamtuf@coredump.cx 6th Apr 2016.
+    inreplace "Makefile", "all: test_x86 $(PROGS) afl-as test_build all_done", "all: test_x86 $(PROGS) afl-as all_done"
```

Comment:
Use a block for `inreplace` when you're doing the same file multiple times:

```ruby
inreplace "Makefile" do |s|
  s.gsub! "ducks", "dogs"
  s.gsub! "wine", "beer"
end
```
And such.